psydac/pyccel/ast/core.py

```diff
@@ -3413,8 +3413,8 @@ def __init__(
 
         # arguments
 
-        if not iterable(arguments):
-            raise TypeError('arguments must be an iterable')
+        #if not iterable(arguments):
+        #    raise TypeError('arguments must be an iterable')
```

Comment:
What is the reason for this change?

Reply:
Sphinx imports all of the files (and at least also those imports of each file that are psydac modules) for which documentation is generated. When trying to document several different files, every time I got a similar error message, ending with this particular TypeError. 

When trying to import said files in ipython, this TypeError was not raised :man_shrugging:

Reply:
All right. Can you please add a dated comment to the code?

Reply:
dd499a8917e7f04f6a736fb765b69224630f574e